# Publish all joint states -----------------------------------
joint_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: 50

# Position Controllers ---------------------------------------
BaseLink_Leg1_joint_position_controller:
  type: "position_controllers/JointPositionController"
  joint: Base_Link_Leg_1
  pid: {p: 1.0, i: 0.0, d: 0.0}

BaseLink_Leg2_joint_position_controller:
  type: "position_controllers/JointPositionController"
  joint: Base_Link_Leg_2
  pid: {p: 1.0, i: 0.0, d: 0.0}

BaseLink_Leg3_joint_position_controller:
  type: "position_controllers/JointPositionController"
  joint: Base_Link_Leg_3
  pid: {p: 1.0, i: 0.0, d: 0.0}

BaseLink_Leg4_joint_position_controller:
  type: "position_controllers/JointPositionController"
  joint: Base_Link_Leg_4
  pid: {p: 1.0, i: 0.0, d: 0.0}

BaseLink_Leg5_joint_position_controller:
  type: "position_controllers/JointPositionController"
  joint: Base_Link_Leg_5
  pid: {p: 1.0, i: 0.0, d: 0.0}

BaseLink_Leg6_joint_position_controller:
  type: "position_controllers/JointPositionController"
  joint: Base_Link_Leg_6
  pid: {p: 1.0, i: 0.0, d: 0.0}

World_BaseLink_joint_position_controller:
  type: "position_controllers/JointPositionController"
  joint: world_Base_Link
  pid: {p: 1.0, i: 0.0, d: 0.0}

Leg1_wheel1_joint_position_controller:
  type: effort_controllers/JointPositionController
  joint: leg1_wheel1
  pid: {p: 1.0, i: 0.0, d: 0.0}

Leg2_wheel2_joint_position_controller:
  type: effort_controllers/JointPositionController
  joint: leg2_wheel2
  pid: {p: 1.0, i: 0.0, d: 0.0}
